src/lang/en-US.ts

```diff
@@ -294,6 +299,12 @@ const EntityTable: ResolvedIntlConfig['messages'] = {
 
     'entityTable.technicalDifficulties.header': `There was an issue getting your data.`,
     'entityTable.technicalDifficulties.message': `We apologize for the inconvenience. You'll find a message describing the issue at the top of the page.`,
+
+    'optionMenu.ariaLabel': `Open Option Menu`,
+    'optionMenu.tooltip': `Options`,
+    'optionMenu.option.detailsPanel.display': `View details`,
+    'optionMenu.option.detailsPanel.hide': `Hide details`,
+    'optionMenu.option.edit': `Edit catalog`,
```

Comment:
Change to "Edit specification."
(We recently re-defined some terminology to make discussing concepts more straightforward. "Catalog" now refers collectively to the group of entities that exist on the control plane; the files that describe it should be referred to as "specifications," "Flow specifications," "Capture specifications," etc)

Reply:
Updated.